### ISSUE TITLE
[Question] Is there a way to view the entire subtree with one click on a folder

### ISSUE THREAD

**Issue by shankarswamy** [omeryanar/FileExplorer#79]
What I would like to do is click on a particular folder or subfolder and have it expand all the way to the leaf nodes.

Is there a way to do it or, if you are familiar with the codebase, could you give me some pointers on how add that.

Thank you!

**Comment by shankarswamy** [comment]
Thank you!!!

**Comment by shankarswamy** [comment]
Hmm .... on my Windows 11 system does not seem to work.  The "Expand" menu shows up; but when I click on a particular level, or select Ctrl + 1 or 2 or ... any of them, nothing happens.

Is there a prerequisite I am missig? I am with the latest .NET and Windows 11 with all latest patches as of now.

**Comment by omeryanar** [comment]
No, I don't think it is a .NET issue. Can you try it by selecting a folder that doesn't contain too many items? If there are too many folders (like program files) it may take too long to load them all.
If still nothing happens, can you look at the Log.txt file in the location of FileExplorer.exe?
